src/reducers/waypoint-reducer.ts

```diff
@@ -0,0 +1,99 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import type { AppDispatch } from "../store";
+import * as Location from "expo-location";
+import * as Cellular from "expo-cellular";
+import { sendNewWaypoint } from "../requests";
+
+interface Waypoint {
+	route_id: string;
```

Comment:
Let's use `camelCase` for everything in TypeScript — it's clearer to not have different name conventions in the code.

Reply:
Fixed